static/js/app.js

```diff
@@ -2,7 +2,7 @@
 
 function dismissNotification() {
 	const notification = document.querySelector(".js-notification");
-	notification.classList.toggle("is-hidden");
+	notification.classList.add("is-hidden");
```

Comment:
The usage of `toggle` here created an edge case where if the code runs when its hidden, it'll become visible Replaced to have explicit HIDE ME properties.

Reply:
I'd say avoiding toggling code is a good thing in general, since it can often lead to unexpected behaviour when e.g. a user quickly hits the same button multiple times. In other words: nice!